src/expressions/if-expr.md

```diff
@@ -106,11 +106,26 @@ is equivalent to
 
 ```rust,ignore
 match EXPR {
-    PAT => { /* body */ },
+    PATS => { /* body */ },
     _ => { /* else */ },    // () if there is no else
 }
 ```
 
+Multiple patterns may be specified with the `|` operator. This has the same semantics
```

Comment:
```suggestion
Multiple patterns may be specified with the `|` operator. A leading `|` is also allowed. This has the same semantics
```

Reply:
I'm not sure I agree with this change. I think it is clear from the grammar that a leading `|` is allowed.  In general I'd like to reduce the amount of "describe the grammar in prose" unless it is necessary.  I can add it if you feel strongly about it, but I would lean towards leaving this out.

Reply:
Hah. I actually agree with your sentiment but wanted to be sure you didn't miss anything... I also lean towards leaving it out.